chore: Update Contributing doc with actions & readme with quick links (#1337)
block/goose: CONTRIBUTING.md

# Contribution Guide

Goose is Open Source! There are numerous ways to be an open source contributor and contribute to Goose. We're here to help you on your way! Here are some suggestions to get started. If you have any questions or need help, feel free to reach out to us on [Discord](https://discord.gg/block-opensource).

> - **Stars on GitHub:** If you resonate with our project and believe in open source, consider starring our Goose on GitHub! 🌟

Reviewer: ```suggestion
- **Stars on GitHub:** If you resonate with our project and find it valuable, consider starring our Goose on GitHub! 🌟
```

- **Ask Questions:** Your questions not only help us improve but also benefit the community. If you have a question, don't hesitate to ask it on [Discord](https://discord.gg/block-opensource).

> - **Give Feedback:** Have a feature you want to see or encounter an issue with Goose, [click here to open an issue](https://github.com/block/goose/issues/new), [start a discussion](https://github.com/block/goose/discussions) or tell us on Discord.

Reviewer: ```suggestion
- **Give Feedback:** Have a feature you want to see or encounter an issue with Goose, [click here to open an issue](https://github.com/block/goose/issues/new/choose), [start a discussion](https://github.com/block/goose/discussions) or tell us on Discord.
```
